Fixed problem with man page, option --keep/-k has proper description now
--- a/pxz.1
+++ b/pxz.1
@@ -24,7 +24,7 @@
 Write results to standard output.
 .TP
 .I "\-k," "\-\-keep"
-Write results to standard output.
+Keep (don't delete) input files
 .TP
 .I "\-T," "\-\-threads"
 Maximal number of threads to run simultaneously.
